spring-security-mvc-socket/src/main/java/com/baeldung/springsecuredsockets/controllers/SocketController.java

```diff
@@ -19,10 +27,20 @@
     private SimpMessagingTemplate simpMessagingTemplate;
     private static final Logger log = LoggerFactory.getLogger(SocketController.class);
 
-    @MessageMapping("/secured/chat")
-    @SendTo("/secured/history")
-    public OutputMessage send(Message msg) throws Exception {
+    @MessageMapping(SECURED_CHAT)
+    @SendTo(SECURED_CHAT_HISTORY)
+    public OutputMessage sendAll(Message msg) throws Exception {
         OutputMessage out = new OutputMessage(msg.getFrom(), msg.getText(), new SimpleDateFormat("HH:mm").format(new Date()));
         return out;
     }
+
+    /**
+     * Example of sending message to specific user using 'convertAndSendToUser()' and '/queue'
+     */
+    @MessageMapping(SECURED_CHAT_ROOM)
+    //@SendToUser(SECURED_CHAT_SPECIFIC_USER)
```

Comment:
Is this line commented out intentionally? If so, is there a reason to leave the line in there, or could we just delete it?